### ISSUE TITLE
 Incorporate haptics on homescreen loading, success and error states

### ISSUE THREAD

**Issue by wavelaurenrussell** [department-of-veterans-affairs/va-mobile-app#8643]
### Description
We want to incorporate haptics on the various load/success states that are included in the sections. Any rules set up with the Alert component should be applied here.

Sections to affect: 

- Errors for Activity and About you
- Errors for Category 
        - Error (at bottom, if applicable)

### ACs

- [ ] Haptics are enabled and trigger with the various states.

**Comment by TKDickson** [comment]
Using karsyn and judy in staging - if I have a haptic that I should have (home/category), and then switch to another tab using the bottom nav bar (payments is the best one to use, since you can be certain that you should NEVER have a haptic on that screen), it's still activating the haptic on switching, somehow. 

Shorter version - we're doing haptics when we ought not to, something to do with how we're navigating the stack/moving around with the bottom tab bar.

**Comment by TKDickson** [comment]
Binny asked a clarifying question in slack, so specifying the behavior a little more:
Home fully loads, haptics fire for home  (great)
The switch to payments, at least one haptic goes again (not good)
Switch to a tab with just stuff loading, haptics goes again, etc (also not good)

**Comment by TKDickson** [comment]
Flicker is fixed, and didn't reintroduce the 'haptics firing on non-haptics stacks' thing I had originally found either. Haptics still firing on home & category screens when valid. Good to merge to feature branch.

**Comment by TKDickson** [comment]
Merged; closing
